src/AppBundle/Controller/BundleController.php

```diff
@@ -100,7 +100,7 @@ public function showBundlesListAction(Request $request)
         $sortingForm = $this->formFactory->create(OrderType::class);
         $searchForm = $this->formFactory->create(SearchType::class);
 
-        $order = null;
+        $order = 'default';
```

Comment:
What is `default`? Is it the default order of the bundle list (when you access the first time the page)?

Reply:
The `order` can't be `NULL` because based on this we are setting-up order in the query, see:
https://github.com/ezsystems/ezplatform-com/blob/master/src/AppBundle/QueryType/BundlesQueryType.php#L47
It was the reason of duplicated entries on the front-end bundle list.